gems/sorbet/lib/fetch-rbis.rb

```diff
@@ -19,7 +19,7 @@ class Sorbet::Private::FetchRBIs
   XDG_CACHE_HOME = ENV['XDG_CACHE_HOME'] || "#{ENV['HOME']}/.cache"
   RBI_CACHE_DIR = "#{XDG_CACHE_HOME}/sorbet/sorbet-typed"
 
-  SORBET_TYPED_REPO = 'https://github.com/sorbet/sorbet-typed.git'
+  SORBET_TYPED_REPO = ENV['SRB_SORBET_TYPED_REPO'] || 'https://github.com/sorbet/sorbet-typed.git'
```

Comment:
I think we're more or less ok with the idea, but this implementation has problems.

If the `RBI_CACHE_DIR` already exists, it will not re-clone the sorbet-typed repo using what's in the env var.

The code should probably detect the conflict and error, asking the user to delete the cache.
If you can think of a better way to handle this case, let me know.
The situation to avoid is to silently do the wrong thing.

Reply:
Good callout. For now I think your suggestion is the easiest thing to do so I've done just that. See the updated PR description with the manual testing I did using this branch.